exporter/coralogixexporter/factory.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coralogixexporter
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// NewFactory by Coralogix
+func NewFactory() component.ExporterFactory {
+	return exporterhelper.NewFactory(
+		typestr,
+		createDefaultConfig,
+		exporterhelper.WithTraces(createTraceExporter),
+	)
+}
+
+func createDefaultConfig() config.Exporter {
+	return &Config{
+		ExporterSettings: config.NewExporterSettings(config.NewComponentID(typestr)),
+		QueueSettings:    exporterhelper.DefaultQueueSettings(),
+		RetrySettings:    exporterhelper.DefaultRetrySettings(),
+
+		Endpoint:   "",
+		PrivateKey: "",
+		AppName:    "",
+		SubSystem:  "",
+	}
+}
+
+func createTraceExporter(_ context.Context, set component.ExporterCreateSettings, config config.Exporter) (component.TracesExporter, error) {
+	// cfg := config.(*Config)
+	set.Logger.Info("Coralogix traces exporter configuration started")
```

Comment:
The Collector already logs when a component is created or initialized, so this log is not necessary